src/routes/safe/components/Transactions/helpers/EditTxParametersForm/index.tsx

```diff
@@ -122,11 +126,12 @@ export const EditTxParametersForm = ({
       <Block className={classes.container}>
         <GnoForm
           initialValues={{
-            safeNonce: safeNonce || 0,
+            safeNonce: safeNonce || '0',
```

Comment:
Why are you falling back to 0. Should we not use the recommended nonce here?

Reply:
I will keep this comment active as a reminder to use the hook that was created to fetch the recommended nonce that should be now on `dev`